projects/kalidea/kaligraphi/src/lib/molecules/kal-datepicker/autofocus.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, ElementRef, Input, OnInit } from '@angular/core';
+
+@Directive({
```

Comment:
this directive could be usefull in another components. Should be global and placed in an UtilityModule or whatever name is best